DllImportGenerator/DllImportGenerator/ContiguousCollectionElementMarshallingCodeContext.cs

```diff
@@ -12,14 +12,14 @@ namespace Microsoft.Interop
 {
     internal sealed class ContiguousCollectionElementMarshallingCodeContext : StubCodeContext
     {
-        private readonly string indexerIdentifier;
         private readonly string nativeSpanIdentifier;
-        private readonly StubCodeContext parentContext;
 
         public override bool SingleFrameSpansNativeContext => false;
 
         public override bool AdditionalTemporaryStateLivesAcrossStages => false;
 
+        public string IndexerIdentifier { get; }
```

Comment:
`init;`?

Reply:
This property is calculated internally, so we don't want external code that creates `ContiguousCollectionElementMarshallingCodeContext` instances to assign it.

Reply:
Oh. I didn't realize that `init` was also subject to scope - I thought it was limited to constructor.